README.md

```diff
@@ -17,7 +17,8 @@ and services such as requests routing, authentication, rate limiting, etc.
 | 3.0.x        | >= 0.9, <0.12  | >= 6.0.0     |
 | 3.1.x, 3.2.x | >= 0.9, <0.13  | >= 6.0.0     |
 | 3.3.x, 3.4.x | >= 0.9, <0.14  | >= 6.0.0     |
-| 3.5.x        | >= 0.9, <0.15  | >= 6.0.0     |
+| 3.5.x        | >= 0.9, <0.15  | >= 6.0.0     |    
+| 3.6.x        | >= 0.9, <1.1.x | >= 6.0.0     |    
```

Comment:
@illyaMs First of all, thanks for this great work. And sorry for taking so time to respond... 

I believe Kong is following semver, so it should be safe to assume that the next version of Kong dashboard that you produced will be compatible with Kong < 2.0.0 and not just < 1.1.0.

Also, I was considering making it a major Kong-dashboard release (4.0.0) and dropping support for Kong < 1.0. The codebase, is becoming more and more difficult to read, especially tests. Now that Kong should be stable, I think it's time to stop supporting the 0.x versions. What do you think?

Reply:
@PGBI , no problem, nice to see you're still here 👍 

Agree, I will change the `1.1.0` hardcodes to `2.0.0` everywhere and commit right now.

Speaking of dropping the pre-1.0.x support - I can only completely agree. Was thinking of it each time while adding one more hard-coded version check in tests, but obviously wasn't able to make this decision on my own :)

May be we can leave the `>= 0.9, <2.0.0` support for v3.6.0 (to deliver existing work done with minimal effort and unblock the community to use the dashboard right away)? And agree to have a next 4.0.0 release with a major cleanup (so me or anyone else who's capable to commit on that part in the nearest future may have their time to work on that)?

Reply:
sounds good to me. Will do the merge and release this weekend. Thanks for your patience.

Reply:
I installed kong1.0.3 and this dashboard 3.6.0. The left menu are OK. But the comsumer's oauth2 plugin still not work well. The return url could not be set.

Reply:
> I installed kong1.0.3 and this dashboard 3.6.0. The left menu are OK. But the comsumer's oauth2 plugin still not work well. The return url could not be set.

Some of the plugins were tested for compatibility, but I don't remember exactly if the oauth2 one was on the list. I think there were some Unit Tests covering it, so actually I think it should be.

Could you provide more details (ideally in separate github issue, linked to this MR/issue)?